### ISSUE TITLE
NC26 App does not load

### ISSUE THREAD

**Issue by b90g** [powerpaul17/nc_money#45]
NC26
php8.2-fpm
mariadb 10.6
debian
apache2

Browser Messages:

Firefox 111
```
Uncaught ReferenceError: process is not defined

https://REDACTED/apps/money/js/money-main.iife.js?v=6055bb5d-59:5
https://REDACTED/apps/money/js/money-main.iife.js?v=6055bb5d-59:5610
```
Chromium:
```
Uncaught ReferenceError: process is not defined
    at money-main.iife.js?v=6055bb5d-59:5:3143
    at money-main.iife.js?v=6055bb5d-59:5610:6910
```

No Plugins or anthing.

**Comment by myacas** [comment]
I've the same issue

NC26
PHP 8.1.17
mysql 10.8.7
OS Linux 5.10.0-15-amd64 x86_64
nginx

Browser MS Etch

` Uncaught ReferenceError: process is not defined
    at money-main.iife.js?v=e75be11a-18:5:3143
    at money-main.iife.js?v=e75be11a-18:5610:6910
(anonym) @ money-main.iife.js?v=e75be11a-18:5
(anonym) @ money-main.iife.js?v=e75be11a-18:5610
globals.js:59 `

**Comment by powerpaul17** [comment]
Thanks for the report. Although I currently do not yet support NC26, and I'm not completely sure why this doesn't happen on NC25, there seems to be an underlying problem. I'll have a look into it later..

**Comment by mostvexingparse** [comment]
I get the same Error after I updated my NC to 25.0.5.1

Error message in Firefox 111:
```
Uncaught ReferenceError: process is not defined
    <anonymous> https://REDACTED/apps/money/js/money-main.iife.js?v=4354a7e4-0:5
    <anonymous> https://REDACTED/apps/money/js/money-main.iife.js?v=4354a7e4-0:5610
money-main.iife.js:5:3129
```

**Comment by powerpaul17** [comment]
After the upgrade, I can confirm that the error happens on 25.0.5 too..

**Comment by powerpaul17** [comment]
Fixed in 83e4d49fe5cc5d1ebd0e847d8ff3824de2a3eebc
